agenthub/codeact_agent/codeact_agent.py

```diff
@@ -193,6 +226,11 @@ def step(self, state: State) -> Action:
             ],
             temperature=0.0,
         )
+        cur_cost = completion_cost(completion_response=response)
+        self.cost_accumulator += cur_cost
```

Comment:
Nice! We definitely need to track this in llm.py eventually, along with the token counts